components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Data Feeds"
+description = "Exporting Node Data from Chef Client Runs to External Data Collectors"
+date = 2020-05-05T13:19:02-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "settings"
+    weight = 10
++++
+
+## About Automate Data Feeds
+
+The Data Feed service is used to send node data to a 3rd party service. This can be useful when updating Configuration Management Databases, external Security Dashboards and ITSM platforms.
+The following types of information are sent:
+ 
+ - Ohai data gathered from each node being managed, this includes HW, OS and installed program information, there is some variation depending on the OS being managed
+ - Configuration information about each node being managed, this includes Chef Client Run status, Runlists, Cookbooks, Recipes being run against each node
+ - Compliance information about each node that shows the compliance state, this includes passed and failed controls for each profile executed against that node
+
+A Data Feed is not a Data Tap, it operates by doing the following:
+
+- Every 4 hours (configurable) the data-feed-service will aggregate the client runs and compliance reports from the previous 4 hours and send this to the registered destinations.
+- If there are no destinations, aggregation will not occur.
+- The data is aggregated and sent in batches of 50 nodes at a time (also configurable)
+
+
+By default only Admins of Chef Automate may create and manage Data Feeds.
+
+### Adding a Data Feed Endpoint
+
+To add an endpoint for a Data Feed in Chef Automate:
+
+![Setup Data Feed Page](/images/docs/filled_form_create_data_feed.png)
+
+1. In the **Settings** tab, navigate to the _Data Feeds_ page in the sidebar.
+1. Select **Create Data Feed**.
+1. Enter a unique Data Feed name.
+1. Enter the URL for your Data Feed End Point, including any specific port details
+1. Enter the Username and Password that your 3rd party endpoint requires for Authentication
+1. Use the **Test Data Feed** button to validate the connection details
+1. Once the Test is successful, use the **Create Data Feed** button to save your Data Feed Configuration
+
+### Edit a Data Feed Endpoint
+
+To edit an Endpoint for a Data Feed in Chef Automate:
+
+1. From the _Data Feeds_ page, select the Data Feed name to open its detail page.
+1. Edit the Data Feed name or URL.
+1. Use the **Save** button to save the Data Feed.
+
+### Delete a Data Feed Endpoint
+
+To delete an Endpoint for a Data Feed in Chef Automate:
+
+1. From the _Data Feeds_ page, select **Delete Data Feed** from the menu at the end of the table row.
+1. Confirm that you wish to permanently delete this Data Feed by using the **Delete Data Feed** button.
+
+## Configuring Global Data Feed Behaviour
```

Comment:
@davymcaleer Is Data Feed set up to appear automatically to users after a certain version, or does this section describe turning on the Data Feed feature?